app/controllers/support_interface/provider_users_controller.rb

```diff
@@ -5,10 +5,10 @@ def index
         .includes(providers: %i[training_provider_permissions ratifying_provider_permissions])
         .page(params[:page] || 1).per(30)
 
+      @filter = SupportInterface::ProviderUsersFilter.new(params: params)
+
       @provider_users = scope_by_use_of_service
       @provider_users = scope_by_search_term
```

Comment:
Can these be amended to accept filter as a param so it's clear why the filter needs to be initialized before they are called.

Reply:
Good point, fixed 👍